docs/dev/stream/operators/process_function.md

```diff
@@ -246,10 +246,8 @@ class CountWithTimeoutFunction extends KeyedProcessFunction[Tuple, (String, Stri
 </div>
 </div>
 
-{% top %}
```

Comment:
This is useful to scroll to the page top. You can find the top link is used in many places.